packages/apps/plugins/plugin-markdown/src/components/DocumentSection.tsx

```diff
@@ -20,25 +28,32 @@ const DocumentSection: FC<{
   const { t } = useTranslation(MARKDOWN_PLUGIN);
   const identity = useIdentity();
   const space = getSpaceForObject(document);
-  const model = useTextModel({ identity, space, text: document?.content });
-  if (!model) {
-    return null;
-  }
 
-  return (
-    <MarkdownEditor
-      model={model}
-      extensions={extensions}
-      placeholder={t('editor placeholder')}
-      slots={{
-        root: {
-          className: mx('flex flex-col grow m-0.5 min-bs-[8rem]', attentionSurface, focusRing),
-          'data-testid': 'composer.markdownRoot',
-        } as HTMLAttributes<HTMLDivElement>,
-        editor: {
-          className: 'h-full py-4',
+  const { themeMode } = useThemeContext();
+  const { doc, accessor } = useDocAccessor(document.content);
+  const { parentRef } = useTextEditor({
```

Comment:
usage of hook here vs `TextEditor` component? is the assumption that tabster and other stuff in `TextEditor` is only needed for the main editor?